.changeset/eighty-monkeys-mate.md

```diff
@@ -0,0 +1,5 @@
+---
+'@soramitsu-ui/theme': patch
+---
+
+**feat**: add two new util tokens
```

Comment:
It would be not excessive to note which tokens exactly.

Reply:
done